package.json

```diff
@@ -66,86 +66,83 @@
     "@ionic-native/keyboard": "^5.26.0",
     "@ionic-native/local-notifications": "^5.28.0",
     "@ionic-native/mobile-accessibility": "^5.28.0",
+    "@ionic-native/network": "^5.36.0",
     "@ionic-native/splash-screen": "^5.26.0",
     "@ionic-native/status-bar": "^5.26.0",
     "@ionic-native/vibration": "^5.32.1",
     "@ionic-native/web-intent": "^5.28.0",
-    "@ionic/angular": "^4.11.10",
+    "@ionic/angular": "^5.9.4",
     "@ionic/storage": "2.2.0",
     "@logisticinfotech/ionic4-datepicker": "^1.4.4",
-    "angular-svg-round-progressbar": "^3.0.1",
+    "angular-svg-round-progressbar": "^6.0.1",
     "autoprefixer": "^9.8.6",
-    "avro-js": "^1.11.0",
+    "avro-js": "^1.11.1",
     "base64-js": "^1.5.1",
     "browserify": "^16.3.0",
     "browserify-zlib": "^0.2.0",
     "buffer": "^6.0.3",
     "com-darryncampbell-cordova-plugin-intent": "^2.0.0",
-    "cordova-android-support-gradle-release": "^3.0.1",
+    "compare-versions": "^6.0.0-rc.1",
```

Comment:
do we need to use rc?

Reply:
I think this was automatically installed when I installed the `latest` tag. I'll install the non-rc version.